src/screens/opportunities/opportunities.tsx

```diff
@@ -49,6 +49,9 @@ const OpportunitiesScreen: React.FC = () => {
           </View>
         )}
       />
+      {Boolean(!opportunities?.length) && (
```

Comment:
This expression will always return boolean
```suggestion
      {!opportunities?.length && (
```